wsimod/nodes/wtw.py

```diff
@@ -94,6 +94,53 @@ def __init__(
         self.treated = self.empty_vqip()
         self.liquor = self.empty_vqip()
         self.solids = self.empty_vqip()
+    
+    def calculate_volume(self):
+        """Calculate the volume proportion of treated water.
+
+        Returns:
+            (float): Volume of treated water
+        """
+        return 1 - self._percent_solids - self._liquor_multiplier["volume"]
+    
+    @property
+    def percent_solids(self):
+        return self._percent_solids
+    
+    @percent_solids.setter
+    def percent_solids(self, value):
+        self._percent_solids = value
+        self.process_parameters["volume"]["constant"] = self.calculate_volume()
+
+    @property
+    def liquor_multiplier(self):
+        return self._liquor_multiplier
+    
+    @liquor_multiplier.setter
+    def liquor_multiplier(self, value):
+        self._liquor_multiplier.update(value)
+        self.process_parameters["volume"]["constant"] = self.calculate_volume()
+    
+    def apply_overrides(self, overrides = Dict[str, Any]):
+        """Apply overrides to the process parameters and liquor multipliers.
+
+        Args:
+            overrides (Dict[str, Any]): Dict describing which parameters should
+                be overridden (keys) and new values (values). Defaults to {}.
+        """
+        self.percent_solids = overrides.pop("percent_solids", 
+                                            self._percent_solids)
+        self.liquor_multiplier = overrides.pop("liquor_multiplier", 
+                                               self._liquor_multiplier)
+        process_parameters = overrides.pop("process_parameters", {})
+        for key, value in process_parameters.items():
+            self.process_parameters[key].update(value)
+        
```

Comment:
@barneydobson Would you consider a situation where originally no process_parameters set for pollutant A but then the users want to append a set of process_parameters for the pollutant A? In this case, a Keyerror will be raised.